queries.sql

```diff
@@ -35,4 +35,129 @@ SELECT *
 SELECT *
 	FROM "VET1".animals
 	WHERE
- 	weight_kg >=10.4 AND weight_kg <=17.3;
\ No newline at end of file
+ 	weight_kg >=10.4 AND weight_kg <=17.3;
+
+
+//PART2
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'unspecified';
+
+SELECT * FROM "VET1".animals
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'unspecified';
+
+SELECT * FROM "VET1".animals
+
+ROLLBACK;
+
+SELECT * FROM "VET1".animals
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'digimon'
+	WHERE name LIKE '%mon';
+	
+UPDATE "VET1".animals
+	SET species = 'pokemon'
+	WHERE species IS NULL;
+	
+SELECT * FROM "VET1".animals
+
+COMMIT;
+
+SELECT * FROM "VET1".animals
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'unspecified';
+
+SELECT * FROM "VET1".animals
+
+ROLLBACK;
+
+SELECT * FROM "VET1".animals
+
+START TRANSACTION
+
+UPDATE "VET1".animals
+	SET species = 'digimon'
+	WHERE name LIKE '%mon';
+	
+UPDATE "VET1".animals
+	SET species = 'pokemon'
+	WHERE species IS NULL;
+	
+SELECT * FROM "VET1".animals
+
+COMMIT;
+
+SELECT * FROM "VET1".animals
```

Comment:
- [ ] Great job on your progress with the database queries! 👏 However, I've noticed a recurring issue in your code. 🧐 You've duplicated several queries throughout your codebase, which can make it challenging to read and maintain.

   - Take a look at the highlighted sections in your code; you'll notice that these queries have already been written above but are repeated here. This redundancy not only makes your code confusing but also less efficient. It's important to strive for clarity and maintainability in your code to ensure that it remains easy to understand and update in the future.

   - Consider refactoring your code to eliminate these duplicate queries. This will not only improve code readability but also enhance the overall quality of your application. 🚀

Happy coding! 😊🚀🍀